### DISCUSSION TITLE
VSCode编译提示报错

### DISCUSSION THREAD

**Discussion by JasonZhang517** [sjtug/SJTUThesis#740] [提问]
在Windows VScode中使用Recipe:latexmk进行编译
提示latexmk: The Perl interpreter could not be found.
但是我已经在电脑中安装了Perl interpreter,使用的是这个网址:https://www.activestate.com/products/perl/
重启vscode还是报这个错,请问有什么解决办法吗

**Comment by JasonZhang517** [comment]
用的是Tex Live

**Comment by LogCreative** [comment]
可能需要把 perl 加入到系统路径里，命令行里直接输入 `perl -v` 能够检测到才可以
